client/src/components/userMenu/userMenu.sass

```diff
@@ -30,6 +30,7 @@
 	margin-right: 5px
 	box-shadow: 0 0 8px #a9a9a9
 	padding: 1px 0
+	z-index: 99
```

Comment:
z-index is a slippery slope

Reply:
What would suggest in place of it? It was going behind the cards before that.

Reply:
Sometimes you need to, but its often the fault of badly structured html